x-pack/plugins/cases/server/client/cases/search.ts

```diff
@@ -57,39 +63,61 @@ export const find = async (
       licensingService.notifyUsage(LICENSING_CASE_ASSIGNMENT_FEATURE);
     }
 
+    /**
+     * Verify if custom field type is filterable
+     */
+    if (paramArgs?.customFields && !isEmpty(paramArgs?.customFields)) {
+      if (configurations.length) {
+        customFieldsConfiguration = paramArgs.owner
+          ? configurations[0].customFields
```

Comment:
why is it `0` if there is no owner?

Reply:
It is `0 `when there is owner.  😄 
If we have owner in params, `await casesClient.configure.get(configArgs)` returns only one configuration.
If we don't have owner, it returns all three configurations.

Reply:
Ah stupid me you're right 😆